File: CSharp/Remove_Element.cs
public class Solution {
    public int RemoveElement(int[] nums, int val) {
        int x = 0;
        for(var i = 0;i < nums.Length;i++){
            if(nums[i] != val){
                nums[x] = nums[i];
                x++;
            }
        } return x;
    }
}
// 0 1 3 0 4 0 4 2  

// https://leetcode.com/problems/remove-element/description/?envType=problem-list-v2&envId=niqxdky2

// 27

File: Python/Remove_Element.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        x = 0
        for i in range(len(nums)):
            if nums[i] != val:
                nums[x] = nums[i]
                x += 1
        return x
# 0 1 3 0 4 0 4 2  
